go/common/crypto/signature/signers/ledger/ledger_signer.go

```diff
@@ -0,0 +1,126 @@
+package ledger
+
+import (
+	"errors"
+	"io"
+
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
+
+	ledger "github.com/zondax/ledger-oasis-go"
+)
+
+const (
+	// SignerName is the name used to identify this package.
+	SignerName = "ledger"
+)
+
+var (
+	_ signature.SignerFactoryCtor = NewFactory
+	_ signature.SignerFactory     = (*Factory)(nil)
+	_ signature.Signer            = (*Signer)(nil)
+
+	// SignerDerivationPath is the derivation path used for generating
+	// the signature key on the Ledger device.
+	SignerDerivationPath = []uint32{44, 118, 0, 0, 0}
```

Comment:
Not sure if we want a default path. If not, then the factory can also take a derivation path as an argument, and `roleDerivationPath` would be unnecessary.

Reply:
No, role-based derivation path would still be necessary if you wanted support for multiple roles (currently not, but maybe some day). But each role could just be a suffix to the passed derivation path.

If the derivation path (prefix) is configurable, it should be passed in via configuration arguments.

Reply:
We careful with `118`. That number belongs to Cosmos. I know the SLIP-0044 registration is still missing, but maybe it is a good idea to unify this `118` into a single place to avoid future mix ups.

Reply:
Yeah any kind of default derivation path prefix should be clearly annotated and not use magic numbers. E.g., there should be a constant that defines 44 and 128 (or something else) with comments, so that it is more clear where these come from (e.g., from SLIP-44 registration).